cmd/cgroups-playground/main.go

```diff
@@ -54,5 +57,30 @@ func xmain() error {
 	for i, s := range subsystems {
 		logrus.Infof("Subsystem %d: %q", i, s.Name())
 	}
+
+	cpuCgroup, err := v2.NewCpu(unifiedMountpoint)
+	if err != nil {
+		return err
+	}
+	cpuGroupPath := v2.GroupPath(filepath.Join(unifiedMountpoint, "cpu"))
```

Comment:
Creating "cpu" under "/sys/fs/cgroup" is confusing, it looks as if mixing up v1 and v2